tailwind.config.ts

```diff
@@ -14,6 +15,22 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
+        darkBlue: "#032541",
+        darkGrey: "#757575 ",
+        purple: "#805BE7",
+        lightBlue: "#01B4E4",
+        lightBrown: "#897474",
+        grey: "#D7D7D7",
+        lightGrey: "E4E4E4",
+      },
+    },
+    borderRadius: {
+      sm: " 5px",
+      md: "8px",
+      lg: "16px",
+      xl: "30px",
+      full: "100%",
```

Comment:
WE can name these as 5,8, ...
So that we don't need to remember what values sm has etc.
But again that is personal choice

Reply:
ok changing

Reply:
Done